src/main/java/de/dennisguse/opentracks/viewmodels/IntervalStatisticsModel.java

```diff
@@ -75,5 +75,10 @@ private void loadIntervalStats(final Track.Id trackId, IntervalOption interval)
         public int getValue() {
             return value;
         }
+
+        @Override
+        public String toString() {
+            return "" + value;
```

Comment:
Only for curiosity: why do you use ```"" + value``` instead of ```String.valueOf(value)```? Is it just a preference of syntax or a question of performance?

Reply:
It just look simpler (to me). I hope that the generated bytecode is the same.

Reply:
Thank you @dennisguse for the clarification. I understand, yes it's more simple :+1: